src/helpers/report.cjs

```diff
@@ -138,7 +138,7 @@ class Report {
 		this._reportVersionOriginal = reportVersionOriginal;
 
 		if (reportVersionOriginal < latestReportVersion) {
-			upgradeReport(report);
+			report = upgradeReport(report);
```

Comment:
This was a bug, but since the report was never modified it actually didn't do anything. Fixed it now cause I saw it.